modules/i18n/src/translations/en-US/portals/console.ts

```diff
@@ -12534,7 +12534,8 @@ export const console: ConsoleNS = {
                             placeholder: "Enter a name",
                             requiredErrorMessage: "Name is a required field",
                             validationErrorMessages: {
-                                alreadyExistsErrorMessage: "A user store with this name already exists."
+                                alreadyExistsErrorMessage: "A user store with this name already exists.",
+                                maxCharLimitErrorMessage: "User store name cannot exceed {{maxLength}} characters."
```

Comment:
Is not updating the FR and LK translations intentional?

Reply:
Yes this is an optional property.